packages/ui/components/scrollable/ScrollableArea.tsx

```diff
@@ -35,7 +35,7 @@ const ScrollableArea = ({ children, className }: PropsWithChildren<{ className?:
         className // Pass in your max-w / max-h
       )}>
       {children}
-      {isOverflowingY && <div style={overflowIndicatorStyles} data-testid="overflow-indicator" />}
+      {isOverflowingY && <div data-testid="overflow-indicator" />}
```

Comment:
scrollableArea is not used anywhere else in the codebase, but when I tried to use it the overflow indicator is hiding the content and is not placed in the right place